uni/udb/data.icn

```diff
@@ -89,18 +89,18 @@ method printStatics(level)
    local x, i, CMonitored := DState.coState.target.val, output := [], staticsTable, variablesList := [], consoleMsg := ""
 
    /level := DState.coState.target.curr_frame
-   if \DState.mode ~== "-adapter" then
+   if \DState.mode ~== "--adapter" then
       put (output, "\n")
    put(output,"   Static variables:")
 
    every x := !DState.srcFile.getCurrStatics() do {
       i := variable(x, CMonitored, level)
       put(output,"     "||x||" = "||image(i)||" : "|| type(i))
-      if \DState.mode == "-adapter" then
+      if \DState.mode == "--adapter" then
          put(variablesList, table("name", x, "value", i, "type", type(i)))
    }
 
-   if /DState.mode | \DState.mode ~== "-adapter" then
+   if /DState.mode | \DState.mode ~== "--adapter" then
```

Comment:
Since you have a null check at the beginning, no need to do a no null check again :)

Reply:
That makes too much sense